packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -854,8 +854,12 @@ class EnumerationForm extends React.Component {
 			// checking if the value already exist
 			const valueExist = this.valueAlreadyExist(value, prevState);
 			const [validateAndAddAction, validateAction, abortAction] = prevState.headerInput;
+			if (!validateAction && !abortAction) {
+				return prevState;
```

Comment:
You're in a `setState(fn)`. You should return null instead of current state to indicate that there is no change.
If I'm not wrong, in React 16, returning null doesn't trigger a new render.